app/controllers/dashboards_controller.rb

```diff
@@ -27,15 +26,8 @@ def dashboard
   end
 
   def set_defence_requests
-    if @dashboard.user_role == "solicitor"
-      @active_defence_requests = @dashboard.defence_requests.active
-    else
-      @active_defence_requests = @dashboard.defence_requests
-    end
+    @active_defence_requests = @dashboard.defence_requests.active
     @closed_defence_requests = @dashboard.defence_requests.finished
-  end
```

Comment:
Why are we setting many instance variables on the controller when we have a form object?

I would prefer only setting 1 instance variable in the controller (the form object), with everything the view needs being supplied by the form object